src/Aspire.Hosting/Dashboard/DashboardViewModelService.cs

```diff
@@ -249,9 +247,8 @@ private async Task ProcessServiceChange(WatchEventType watchEventType, Service s
             return;
         }
 
-        foreach (var kvp in _resourceAssociatedServicesMap.Where(e => e.Value.Contains(service.Metadata.Name)))
+        foreach (var ((resourceKind, resourceName), _) in _resourceAssociatedServicesMap.Where(e => e.Value.Contains(service.Metadata.Name)))
```

Comment:
I personally think using _resourceAssociatedServicesMap.Keys would be slightly more readable, but that's personal preference.

Reply:
That's not possible in this case, as it's filtering by value.